### PR TITLE
Fix global geoip config docs.

### PR DESCRIPTION
**Pull request by pillarsdotnet**
The [code](https://github.com/wazuh/wazuh/blob/master/src/config/global-config.c#L255) looks for a [`geoipdb`](https://github.com/wazuh/wazuh/blob/master/src/config/global-config.c#L140) option, but the legacy [`geoip_db_path`](https://github.com/wazuh/wazuh/blob/master/src/config/global-config.c#L165) and [`geoip6_db_path`](https://github.com/wazuh/wazuh/blob/master/src/config/global-config.c#L166) options are unused.

```sh
bobvin@T560:~/github/wazuh egrep -r 'geoip(6?)_db_path *'
src/config/global-config.h:    char *geoip_db_path;
src/config/global-config.h:    char *geoip6_db_path;
src/config/global-config.c:    const char *xml_geoip_db_path = "geoip_db_path";
src/config/global-config.c:    const char *xml_geoip6_db_path = "geoip6_db_path";
src/config/global-config.c:        else if (strcmp(node[i]->element, xml_geoip_db_path) == 0) {
src/config/global-config.c:                os_strdup(node[i]->content, Config->geoip_db_path);
src/config/global-config.c:        else if (strcmp(node[i]->element, xml_geoip6_db_path) == 0) {
src/config/global-config.c:                os_strdup(node[i]->content, Config->geoip6_db_path);
bobvin@T560:~/github/wazuh
```

### CODE REVIEW

**Comment by pillarsdotnet** [comment]
Also see https://github.com/wazuh/wazuh/issues/1489

**Comment by pillarsdotnet** [comment]
Ping?

File: source/user-manual/reference/ossec-conf/global.rst [hunk 1]
```diff
@@ -18,6 +18,7 @@ Options
 -------
 
 - `alerts_log`_
+- `compress_alerts`_
```
Comment (JuanjiJG) [inline source/user-manual/reference/ossec-conf/global.rst]: This setting is not used. Please remove it from reference.

Comment (pillarsdotnet) [reply source/user-manual/reference/ossec-conf/global.rst]: Removed as requested.

**Comment by pillarsdotnet** [comment]
@JuanjiJG -- See latest commit.

**Comment by JuanjiJG** [comment]
Hello @pillarsdotnet,

Thanks for your contribution, we appreciate it.

I'm going to merge this PR into the documentation. It will be publicly available on the next release.

Regards,
Juanjo

**Pull request merged**